### PR TITLE
Demote implicit cradle warn to logging

### PR DESCRIPTION
**Pull request by jneira**

### CODE REVIEW

File: ghcide/session-loader/Development/IDE/Session.hs [hunk 1]
```diff
@@ -395,8 +395,8 @@ loadSessionWithOptions SessionLoadingOptions{..} dir = do
            lfp <- flip makeRelative cfp <$> getCurrentDirectory
            logInfo logger $ T.pack ("Consulting the cradle for " <> show lfp)
 
-           when (isNothing hieYaml) $ mRunLspT lspEnv $
-             sendNotification SWindowShowMessage $ notifyUserImplicitCradle lfp
+           when (isNothing hieYaml) $
+             logWarning logger $ implicitCradleWarning lfp
```
Comment (michaelpj) [inline ghcide/session-loader/Development/IDE/Session.hs:399]: Is this even a *warning*? It seems like info would be plausible: this is something that users mostly won't care about (the message even says that!) but it's useful to have in the logs later if you're trying to figure out what happened.

**Comment by pepeiborra** [comment]
What's the motivation for this change? Is there a related issue or conversation?

File: ghcide/session-loader/Development/IDE/Session.hs (hunk 1 above)
Comment (jneira) [reply ghcide/session-loader/Development/IDE/Session.hs:399]: Agree, there is not much difference given we are handling logs but it can be info

**Comment by jneira** [comment]
> 
> 
> What's the motivation for this change? Is there a related issue or conversation?

Sorry, i've forgotten to link the relevant issues: it would close #620, there you can see the links to other ones, closed as duplicated: #1259, #1229 and https://github.com/haskell/vscode-haskell/issues/326
I've observed several beginners in the discord chat confused with that alert, moreover if they are experiencing some other issue. They dont know nothing about cradles, implicit or explicit and despite the message they tend to think it is related with their issue.

File: ghcide/session-loader/Development/IDE/Session.hs (hunk 1 above)
Comment (jneira) [reply ghcide/session-loader/Development/IDE/Session.hs:399]: I've converted it to info

**Review by pepeiborra — APPROVED** [review]

**Comment by michaelpj** [comment]
Seems like the commit still has it at a warning?

**Comment by ChrisPenner** [comment]
> I've observed several beginners in the discord chat confused with that alert, moreover if they are experiencing some other issue. They dont know nothing about cradles, implicit or explicit and despite the message they tend to think it is related with their issue. (but some of them ask for the warning directly)

For additional context; in neovim with the Coc LSP the warning actually pops up a prompt that you have to manually dismiss. It only pops up once HLS has started initializing, meaning I'm often halfway through typing something and a prompt interrupts me, it's incredibly annoying haha. Very thankful this is fixed 🙌🏼

**Pull request merged**